[CM T1] jni: added example usage for meta keys

mandoway opened the pull request:
<!--
Check relevant points but **please do not remove entries**.
-->

## Basics

<!--
These points need to be fulfilled for every PR.
-->

- [x] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [x] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy. If not, fix **in this order**:
  - [ ] add a line in `doc/news/_preparation_next_release.md`
  - [ ] reformat the code with `scripts/dev/reformat-all`
  - [ ] make all unit tests pass
  - [ ] fix all memleaks
- [x] The PR is rebased with current master.

<!--
If you have any troubles fulfilling these criteria, please write
about the trouble as comment in the PR. We will help you,
but we cannot accept PRs that do not fulfill the basics.
-->

## Checklist

<!--
For docu fixes, spell checking, and similar none of these points below
need to be checked.
-->

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://master.libelektra.org/doc/CODING.md))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

<!--
Reviewers should check the following.
-->

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

<!--
If you are already Elektra developer, please adjust the labels.
Otherwise, write a comment and it will be done for you.
-->

- [ ] Add the "work in progress" label if you do not want the PR to be reviewed yet.
- [ ] Add the "ready to merge" label **if the basics are fulfilled** and no further pushes are planned by you.

## Who did what
@mandoway added java example for meta keys

kodebach commented:
Jenkins build libelektra please

kodebach commented:
Please rebase to fix the conflicts in the release notes.

A tip for the future: When adding the release notes try to leave as much of the file as possible intact, i.e. instead of replacing `<<Binding>>` with `Java` add a new section. This reduces the chance that somebody else changed the same line in another PR.

mandoway commented:
I rebased with master and fixed some formatting issues.

Everytime I rebase and push, github still complains about conflicts. Then I have to fix the same conflict I just fixed again on GitHub.
Is this normal or am I missing something?

kodebach commented:
Currently GitHub doesn't show any conflicts for me. Not sure what the problem was.

Did you maybe forget to pull `master` before rebasing? If you rebase with an outdated `master` you will still see conflicts (if the conflicting change happend after the commit you rebased with).

mandoway commented:
I fetched the repo, rebased onto origin/master, fixed conflicts locally and then pushed.
GitHub showed the changes coming from master.
It also showed a conflict.
I then resolved this conflict (same as local) in the GitHub editor.

It's strange, but now everything is fixed.
